PluginControlInterfaceGeneric/CMakeLists.txt

```diff
@@ -17,9 +19,8 @@ OPTION (NSM_IFACE_PRESENT
 
 SET(PLUGIN_LIB_NAME PluginControlInterfaceGeneric)
 
-SET(INCLUDE_FOLDER "include")
-
-SET(CONF_ROOT "${CMAKE_INSTALL_PREFIX}/etc/controllerconf")
+set(CONTROLLER_CONFIG_DIR "${CMAKE_INSTALL_FULL_SYSCONFDIR}/${LIB_INSTALL_SUFFIX}/control"
```

Comment:
Hello Isogai-san,

Like dealt, this patch should be reverted for this PR, because PR #4 will finally take care of this issue.

Please note that otherwise other patches will not apply correctly!

Thanks in advance for your attention, have nice Holidays!